server/accessmanager_test.go

```diff
@@ -0,0 +1,60 @@
+package server
+import (
+	"testing"
+	"github.com/smancke/guble/guble"
+	"github.com/stretchr/testify/assert"
+	"fmt"
+)
+
+type TestAccessManager struct {
+	access map[string]map[guble.Path]bool
+}
+
+func NewTestAccessManager() *TestAccessManager {
```

Comment:
Using a mock here would have been less effort .. but this is also totally fine for me ..